docs/modules/ROOT/pages/cli/installation-ide.adoc

```diff
@@ -139,6 +148,14 @@ those updates by opening the "BSP" tab and clicking the "Refresh" button
 
 image::basic/IntellijRefresh.png[]
 
+To prepare your project for IDEs, and in general any BSP client, you can also
+run this command to generate the BSP configuration files:
+
+[source,bash]
+----
+./mill mill.bsp.BSP/install
+----
+
```

Comment:
This should not be places under `IntelliJ` section, since it's relevant for _all_ BSP clients.